source/Menu.cpp

```diff
@@ -130,6 +130,7 @@ namespace GTA
 			FooterTextColor,
 			FooterFont,
 			FooterCentered);
+		this->SelectedIndexChanged(this, gcnew SelectedIndexChangedArgs(this->mSelectedIndex));
```

Comment:
This will crash if there is no event assigned to `SelectedIndexChanged`, since it's null